lib/cryptoexchange/exchanges/jex/services/market.rb

```diff
@@ -0,0 +1,57 @@
+module Cryptoexchange::Exchanges
+  module Jex
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            false
+          end
+        end
+
+        def fetch
+          output = super(self.ticker_url)
+          adapt_all(output)
+        end
+
+        def ticker_url
+          "#{Cryptoexchange::Exchanges::Jex::Market::API_URL}"
+        end
+
+        def adapt_all(output)
+          market_pairs = []
+          output.map do |pair, ticker|
+            separator = /( CALL | PUT |ETF)/ =~ pair
```

Comment:
since these are blacklist keywords, maybe call it something other than separator?

Reply:
Noted. I have resolved this issue by creating a method to check derivatives and remove it from the array. Please find the latest PR for your review.